### PR TITLE
Exercise 2: Count decile

### PR DESCRIPTION
**Pull request by nikol1655**

### CODE REVIEW

File: src/Main.java [hunk 1]
```diff
@@ -0,0 +1,45 @@
...
+
+        int index = (int) Math.ceil(decile / 10 * numbers.size());
+        return numbers.get(index - 1);
+    }
+
+    private static List<Long> getSortedListFromUser(long number, String userAnswer) {
+        List<Long> numbers = new ArrayList<>();
+        do {
+            System.out.println("Type the number from 1 to 1000");
+            number = scanner.nextLong();
```
Comment (jaroslawwoloch) [inline src/Main.java]: nextInt is enough.

File: src/Main.java [hunk 2]
```diff
@@ -0,0 +1,45 @@
...
+
+        long number = 0;
+        String userAnswer = "";
+        List<Long> numbers = getSortedListFromUser(number, userAnswer);
+        System.out.println("The 3rd decile is: " + countDecile(numbers, 3));
+        System.out.println("The 6th decile is: " + countDecile(numbers, 6));
+
+    }
+
+    private static long countDecile(List<Long> numbers, double decile) {
```
Comment (jaroslawwoloch) [inline src/Main.java:19]: why static

File: src/Main.java [hunk 3]
```diff
@@ -0,0 +1,45 @@
...
+
+        int index = (int) Math.ceil(decile / 10 * numbers.size());
+        return numbers.get(index - 1);
+    }
+
+    private static List<Long> getSortedListFromUser(long number, String userAnswer) {
```
Comment (jaroslawwoloch) [inline src/Main.java:25]: why static

**Review by jaroslawwoloch — CHANGES_REQUESTED** [review]

File: src/Main.java (hunk 3 above)
Comment (nikol1655) [reply src/Main.java:25]: I cannot call the method  _getSortedListFromUser_ when I delete static: "Non-static method 'countDecile(java.util.List<java.lang.Long>, double)' cannot be referenced from a static context"

File: src/Main.java (hunk 2 above)
Comment (nikol1655) [reply src/Main.java:19]: I cannot call the method _countDecile_  when I delete static: "Non-static method 'countDecile(java.util.List<java.lang.Long>, double)' cannot be referenced from a static context"

**Pull request merged**